accepted/0000-isolated-workspaces.md

```diff
@@ -0,0 +1,177 @@
+- Start Date: 2018-02-12
+- RFC PR: (leave this empty)
+- Yarn Issue: (leave this empty)
+
+# Summary
+
+Running a workspace in isolation should be as easy as running all workspaces together. Running a yarn command in
+a workspace should behave the same whether it is part of a larger project or whether it is its own separate repo. There
+should be a new set of commands, `yarn workspaces <command>` to distinguish between running commands that affect
+the whole workspace from commands that affect a single workspace.
+
+# Motivation
+
+Currently, yarn workspaces are optimized for people making changes across multiple projects in the 
+monorepo, but they can make things more difficult if you want to work on a single workspace in isolation. While
+the automatic linking and auto-installation of dependencies for all workspaces can be convenient if you are actually working
+on all workspaces, having to build and install everything locally for multiple projects when you only work on one
+at a time can be time consuming compared to just pulling those packages down from a registry.
+
+If you could work on a single workspace in isolation as easily as if it were its own repo, the transition to a monorepo
+from separate repos would be seamless for people who focus on a single workspace/repo at a time, and one of the
+biggest reasons not to use workspaces would be removed. 
+
+# Detailed design
+
+Terminology used for this doc:
+- rootDir: the root directory of a workspaces project. the one with the package.json with workspaces in it
+- workspaceDir: Directory for an individual workspace/package
+- regular (isolated) mode: `yarn foo`
```

Comment:
I think isolated is a better term than regular

Reply:
agreed